packages/interface-mocks/src/connection-manager.ts

```diff
@@ -72,7 +77,7 @@ class MockConnectionManager extends EventEmitter<ConnectionManagerEvents> implem
     this.started = false
   }
 
-  getConnections (peerId?: PeerId): Connection[] {
+  getConnections (peerId?: PeerId | Multiaddr[]): Connection[] {
```

Comment:
This method is from the connection manager interface and it doesn't take `Multiaddr[]` - can this be removed?